### DISCUSSION TITLE
State of separate admission webhook

### DISCUSSION THREAD

**Discussion by michalschott** [prometheus-operator/prometheus-operator#5208] [General]
What is current state of this idea? I can see binaries are build and published, but these are not deployed within `bundle.yaml` - is there any specific reason for this?

I want to setup HA for webhooks, and digging through historic discussions looks like this is the way you're looking forward to configure it.

**Comment by simonpasquier** [comment]
The recommended approach is definitely to run a standalone admission webhook deployment. Please have a look at https://prometheus-operator.dev/docs/user-guides/webhook/ and comment if it isn't clear/correct.
The reason why `bundle.yaml` doesn't deploy the standalone webhook is because we depend on tools like cert-manager to provision the TLS certificates all over the place.

**Reply by michalschott** [reply]
Perfect, so is this subproject considered production ready?

**Reply by simonpasquier** [reply]
yes totally
